guardrails/guardrail_globals.py

```diff
@@ -85,19 +91,19 @@ def mutable_lint_cmd(self):
         """ Function to parse and form optional linting command (ignore files and rc file) """
 
         cmd = ""
-        if self.lint_ignore:
-            cmd = "--ignore %s" % self.lint_ignore
         if self.pylintrc:
             cmd = cmd + " " + "--rcfile %s" % self.pylintrc
         return cmd
 
     def generate_pylint_cmd(self):
         """ Function to set optional linting command (ignore files and rc file) """
-        if self.linting:
-            cmd = "%s -m pylint  %s" % (self.python, self.mutable_lint_cmd())
-        else:
-            cmd = ""
-        return cmd
+        cmd_list = []
+        sub_list = [self.generate_files_lint()[i:i + 20] for i in range(0, len(self.generate_files_lint()), 20)]
```

Comment:
What does 20 signify here please?

Reply:
pylint is not working on folders which are not having __init__.py file. Hence taking the approach to to give absolute path to the file to pylint. Here 20 is taking keeping some buffer so that the argument passed will not overflow the buffer

Reply:
To ensure that this doesn't break in the future because of the number 20, here are two options: 
1. Put this 20 as a configuration in the ini file
2. Pass this as an argument to the command line. If not passed, default to 20.

Con for option 1 - is that we'll mix responsibility. So in this case, option 2 looks more promising. Please take a call.

Reply:
Once this is done, please feel free to merge the changes.

Reply:
updated the source to pass it as a command line option